CRFModuleValidation/Cardio_Stress.json

```diff
@@ -0,0 +1,70 @@
+{
+    "identifier"        : "Cardio Stair Step",
+    "actions"           : { "cancel": { "iconName": "closeActivity" }},
+    "shouldHideActions" : ["goBackward", "skip"],
+    "progressMarkers"   : ["heartRate.before", "stressTest", "heartRate.after"],
+    "steps"             : [
+                           {
+                           "identifier"   : "introduction",
+                           "type"         : "instruction",
+                           "title"        : "Cardiovascular stress test",
+                           "text"         : "20 minutes",
+                           "image"        : { "imageName" : "treadmillMan", "placementType" : "topBackground" },
+                           "colorTheme"   : {   "backgroundColor" : "offWhite",
+                                                "foregroundColor" : "gunmetal",
+                                                "usesLightStyle" : false },
+                           "actions"      : {
+                                               "goForward":  { "buttonTitle": "Start" },
+                                               "cancel":     { "iconName": "backArrowHeader" },
+                                               "learnMore":  { "iconName": "infoIcon", "url": "Stress_Test.html" }
+                                            },
+                           "viewTheme"    : { "viewIdentifier": "TaskIntroductionStressTest", "storyboardIdentifier": "ActiveTaskSteps" }
+                           },
+                           {
+                           "identifier"         : "heartRate.before",
+                           "type"               : "transform",
+                           "replacementSteps"   : [{ "identifier"   : "instruction",
+                                                     "title"        : "Capture heart rate",
+                                                     "text"         : "Use your finger to cover the camera and flash on the back of your phone." },
+                                                   { "identifier"   : "feedback",
+                                                     "text"         : "Your heart rate is" }
+                                                   ],
+                           "sectionTransformer"    : { "resourceName": "HeartrateStep.json"}
+                           },
+                           {
+                           "identifier"     : "stressTest",
+                           "type"           : "section",
+                           "steps"          : [
+                                   {
+                                   "identifier"   : "instructions",
+                                   "type"         : "instruction",
+                                   "title"        : "Ready to run on the treadmill?",
+                                   "text"         : "You’ll start off slowly and speed and will be increased through the test duration.",
```

Comment:
"...speed and will..." I think there's a "grade" missing between "and" and "will"? Might also be worth throwing in a comma after "slowly" to help chunk it up for easier parsing.

Reply:
ugh, thanks.